### ISSUE TITLE
The contrast ratio is less than 4.5:1 under Color Options and Status in Badges tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1938]
URL: http://clarity-ks.surge.sh/badges

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Badges tab.
3. Use WAT contrast analyzer to select foreground and background colors from the screen.
4. Check the contrast ratio.

Actual Results:
The contrast ratio is less than 4.5:1 for number 1/15/90/51 under Color Options and number 2/3 under Status.
For more details, please refer to “Badges-LowContrast.png”.

Expected Results:
The contrast ratio is 4.5:1 or greater when comparing all background and foreground colors.

Note:
This issue also occurs on Wizards tab.

Violation of Section 508 Standards: 
Section 1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: Low vision.

Screen shot:
1.	Badges-LowContrast.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/cc4006af-27b8-4717-9bdf-42d81a393750)

**Comment by mathisscott** [comment]
Gray badge ratio = 4.7:1
Purple badge = 4.57:1
Blue badge = 4.54:1
Green badge = 4.57:1

https://webaim.org/resources/contrastchecker/

Some colors (green and gray) have been updated since this ticket was opened. Others, I believe, were called out in error.
